CHANGES.rst

```diff
@@ -4,17 +4,13 @@ Changelog
 1.9.1 (unreleased)
 ------------------
 
-Breaking changes:
-
-- *add item here*
-
-New features:
-
-- *add item here*
-
 Bug fixes:
 
-- *add item here*
+- Fix for dotted path form ids.
+  [malsup]
```

Comment:
Please better make a changelog entry like: ``Updated jquery.form to latest 3.51.0``.
Funny side note, looking up jquery.form on Github I found, that a PR reporting & fixing the dotted path form ids bug was done by me. Couldn't remember that.
It's not even included in mockup, which I might take care of later...